frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -28,20 +28,26 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
   return (
     <Tr>
       <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
-      <Td dataLabel="Experiment">
-        <Link
-          to={
-            isArchived
-              ? experimentArchivedRunsRoute(namespace, experiment.experiment_id)
-              : experimentRunsRoute(namespace, experiment.experiment_id)
+      <Td dataLabel="Experiment" width={40}>
```

Comment:
You can give the `width` in "frontend/src/concepts/pipelines/content/tables/columns.ts" for every column like other tables to get a better layout. Just make sure the sum of all the column widths is 100.